src/components/Navbar/Navbar.js

```diff
@@ -17,18 +30,29 @@ const Navigation = ({ toggleHandler }) => {
           alt="logo"
         ></img>
       </Link>
-      <div className="nav-spacer ">
-        <SearchBar />
-      </div>
+      <button
+        className="set-lang"
+        onClick={(event) => {
+          onClickLang();
+          switchImage();
+        }}
+      >
+        <img
+          className="flag"
+          src={require(`../../assets/img/${flagIMG}`)}
+          alt="flag"
+        />
+      </button>
+      <div className="nav-spacer "></div>
       <ul className="nav-links">
         <Link to="/" className="nav-link">
-          <li>HOME</li>
+          <li className="nav-link-element">{translate("Home")}</li>
         </Link>
         <Link to="/Aboutus" className="nav-link">
-          <li>ABOUT US</li>
+          <li className="nav-link-element">{translate("About")}</li>
         </Link>
         <Link to="/Login" className="nav-link">
-          <li>LOG IN</li>
+          <li className="nav-link-element">{translate("Login")}</li>
```

Comment:
This code seems duplicated in the SideDrawer which is maybe not the best solution